pkg/operator/client/client.go

```diff
@@ -42,9 +44,12 @@ func GetOperatorClient(address, serverName string, certChain *daprCredentials.Ce
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "failed to create tls config from cert and key")
 	}
-	opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(config)))
+	// block for connection
+	opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(config)), grpc.WithBlock())
 
-	conn, err := grpc.Dial(address, opts...)
+	ctx, cancelFunc := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Nit: `30 * time.Second` should be a constant (I don't think we need to make this configurable, yet)

Reply:
Any updates here?

Reply:
Addressed the changes.